src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -289,22 +289,64 @@ public static void notify(final Throwable exception) {
     /**
      * Notify Bugsnag of a handled exception
      *
+     * @param exception the exception to send to Bugsnag
+     * @param callback  callback invoked on the generated error report for
+     *                  additional modification
+     */
+    public static void notify(final Throwable exception, final Callback callback) {
+        getClient().notify(exception, callback);
+    }
+
+    /**
+     * Notify Bugsnag of an error
+     *
+     * @param name       the error name or class
+     * @param message    the error message
+     * @param stacktrace the stackframes associated with the error
+     * @param callback   callback invoked on the generated error report for
+     *                   additional modification
+     */
+    public static void notify(String name, String message, StackTraceElement[] stacktrace, Callback callback) {
+        getClient().notify(name, message, stacktrace, callback);
+    }
+
+    /**
+     * Notify Bugsnag of a handled exception
+     *
      * @param  exception  the exception to send to Bugsnag
      * @param  severity   the severity of the error, one of Severity.ERROR,
      *                    Severity.WARNING or Severity.INFO
+     *
+     * @deprecated Use {@link notify(Throwable,Callback)}
```

Comment:
FYI I kept `notify(Throwable, Severity)` as an option on my rewrite of bugsnag-java for ease of use:
https://github.com/bugsnag/bugsnag-java/blob/next/src/main/java/com/bugsnag/Client.java#L262